File: DataStructures/ExamPreparation-01/Problem-2-PitFortress/C#-Skeleton/PitFortressSkeleton/Classes/Player.cs
namespace Classes
{
    using Interfaces;

    public class Player : IPlayer
    {
        public int CompareTo(Player other)
        {
            throw new System.NotImplementedException();
        }

        public string Name { get; private set; }

        public int Radius { get; private set; }

        public int Score { get; set; }
    }
}

File: DataStructures/ExamPreparation-01/Problem-2-PitFortress/Java-Skeleton/PitFortress/src/main/models/Player.java
package main.models;

import main.interfaces.IPlayer;

public class Player implements IPlayer {

    private int radius;
    private String name;
    private int score;

    @Override
    public String getName() {
        return name;
    }

    @Override
    public int getRadius() {
        return radius;
    }

    @Override
    public int getScore() {
        return score;
    }

    public void setScore(int value) {
        this.score = value;
    }

    @Override
    public int compareTo(Player o) {
       return 0;
    }
}
